code/modules/tables/interactions.dm

```diff
@@ -57,6 +57,65 @@
 			return 1
 	return 1
 
+/obj/structure/table/Crossed(var/atom/movable/am as mob|obj)
+	..()
+	if(ishuman(am))
+		var/mob/living/carbon/human/H = am
+		if(H.a_intent != I_HELP || H.m_intent == "run")
+			throw_things(H)
+		else if(H.is_diona() || H.species.bodytype == "Heavy machine")

```

Comment:
I'm pretty sure you're not supposed to read bodytype directly, there's a helper for this.

Reply:
`Heavy machine` should also be `Heavy Machine`.